src/index.ts

```diff
@@ -0,0 +1,353 @@
+      async (yargs: Argv) => {
+        return yargs
+          .option("deltaAsset", {
+            alias: "a",
+            describe: "the amount of asset to change the position by",
+            type: "string",
+            require: true,
+          })
+          .option("deltaStable", {
+            alias: "s",
+            describe: "the amount of stable to change the position by",
+            type: "string",
+            require: true,
+          })
+          .option("stableBound", {
+            alias: "b",
+            describe: "max price trader is willing to pay",
+            type: "string",
+            default: "0",
+          });
+      },
+      async (argv) => {
+        const { deltaAsset, deltaStable, stableBound } = argv;
+
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        try {
+          const trade = await exchange.callStatic.changePosition(
+            deltaAsset,
+            deltaStable,
+            stableBound
+          );
+
+          console.log({
+            startAsset: trade.startAsset.toString(),
+            startStable: trade.startStable.toString(),
+            totalAsset: trade.totalAsset.toString(),
+            totalStable: trade.totalStable.toString(),
+            tradeFee: trade.tradeFee.toString(),
+            traderPayout: trade.traderPayout.toString(),
+          });
+        } catch (e) {
+          console.log("Can not estimate trade");
+          console.log({ e });
+        }
+      }
+    )
+    .command(
+      ["approveTokens"],
+      "approve_tokens",
+      async (yargs: Argv) => yargs,
+      async (argv: any) => {
+        const { accountNumber, networkId, exchangeAddress } =
+          getStandardParams(argv);
+
+        const wallet = loadAccount(networkId, accountNumber);
+        const exchange = IExchange__factory.connect(exchangeAddress, wallet);
+
+        const assetTokenAddress = await exchange.assetToken();
+
+        const assetToken = IERC20__factory.connect(assetTokenAddress, wallet);
+
+        const tx1 = await assetToken.approve(
+          exchangeAddress,
+          ethers.utils.parseEther("100000")
+        );
+        await tx1.wait();
+
+        const stableTokenAddress = await exchange.stableToken();
+
+        const stableToken = IERC20__factory.connect(stableTokenAddress, wallet);
+
+        const tx2 = await stableToken.approve(
+          exchangeAddress,
+          ethers.utils.parseEther("100000")
+        );
+        await tx2.wait();
+
+        console.log(
+          "Approved both tokens for account: " + (await wallet.getAddress())
+        );
+      }
+    )
+    .command(
+      ["liquidate"],
+      "liquidate",
+      async (yargs: Argv) => {
+        return yargs.option("trader", {
+          alias: "t",
+          describe: "trader",
```

Comment:
We could probably say `describe: "trader address"`.
Or be more verbose and explain it in full:
```javascript
    describe: "Address of a trader that we are trying to liquidate.\n\
Use `estimateLiquidate` to see if this trader position is liquidatable without paying anything.",
```